### PR TITLE
Obligatory NVs and meson googles screen/client color

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
This PR sets `forced_glasses_color` to TRUE for most night vision and meson googles in the game.

The zealot blindfold, as well as the MODsuit component (technical limitations) and NVs from syndicate-aligned outfits and mobs (and the space ninja) are **exempted** from this. The ratio is that it doesn't fit well with the zealot blindfold, while antags such as nuclear operatives and space ninjas tend to wear them throughout entirety of the round if they want to win, so that'd generate an en passe where nukies and the such are forced to get the green light (ba dum tshh) every round, which is mildly undesiderable.

For reference, this is the shade of green from **NVs**. 
immagine
You can compare it with both screenshots for mesons and control from #81328.

**EDIT: After some consideration, I've made it so the colors mesons and NVs gets fainter on lavaland, miners rejoice.**

## Why It's Good For The Game
Since #81328 happened, client screen coloring has generally become much more bearable, especially for the lighter tones like mesons, as shown in the PR itself. About NVs, I believe the green hue should be a cardinal feature of the clichè ~~though real-life NVs are a bit more stark normally, but we've got plenty of cases of people being motion and light sensitive here~~, I rest my case.

TL;DR Muh flavor.

## Changelog

:cl:
add: meson googles and night vision googles now color your screen. Always. Except for nukies and the space ninja.
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
Engineers and miners are pretty much forced to wear mesons for the entire round - this doesn't sound very nice at all for those jobs.

**Comment by Eronymun** [comment]
Engineers actually get yellow meson goggles, which I think have a light yellow hue already? Might be thinking of monke.

Engineers can turn off their mesons when not near the SM, but miners might have a bit of a hard time if the green makes lava land any darker.

**Comment by SmArtKar** [comment]
> **Ok, everyone, I've an Idea! I will make it so the hue from mesons is removed while on lavaland and that of NVs dampened, also on lavaland, with the power of signals. However that'll have to wait a few hours since I'll be a bit busy irl.**
It'd be neat if you also made it possible to turn mesons(other goggles)/NVs off akin to engineering version, so you wouldn't have to physically take them off every time.

**Comment by Cheshify** [comment]
I'm very sorry but I don't know why "Makes Goggles Less Bearable To Use" is a good pr idea. People who like it can already toggle it on.

**Comment by Ghommie** [comment]
> I'm very sorry but I don't know why "Makes Goggles Less Bearable To Use" is a good pr idea. People who like it can already toggle it on.

You're sugarcoating that glasses not having a tint has got so much comfortable with us that even the slightest change becomes a burden, and I hardly believe they care about the toggle at all, which has been broken for some time and no one has made a bug report.

**Comment by alien3301** [comment]
The tint is an option turned off by default for a reason. Almost nobody likes it. It's not like there's an epidemic of mesons and NVs so you need to make them less comfortable to use.

**Comment by Ghommie** [comment]
Eh, I suppose you've made me somewhat change my mind. I'll remove the tint from the mesons and make NVs use the fainter green tint instead (nukies, ninja etc. are still exempted). I'll also go ahead and give NVs a on-off action button.
